packages/flutter_tools/test/ios/code_signing_test.dart

```diff
@@ -211,6 +211,68 @@ void main() {
       AnsiTerminal: () => testTerminal,
     });
 
+    testUsingContext('Test multiple identity in machine mode works', () async {
+      when(mockProcessManager.runSync(<String>['which', 'security']))
+          .thenReturn(exitsHappy);
+      when(mockProcessManager.runSync(<String>['which', 'openssl']))
+          .thenReturn(exitsHappy);
+      when(mockProcessManager.runSync(
+        argThat(contains('find-identity')), environment: any, workingDirectory: any,
+      )).thenReturn(new ProcessResult(
+        1,     // pid
+        0,     // exitCode
+        '''
+1) 86f7e437faa5a7fce15d1ddcb9eaeaea377667b8 "iPhone Developer: Profile 1 (1111AAAA11)"
+2) da4b9237bacccdf19c0760cab7aec4a8359010b0 "iPhone Developer: Profile 2 (2222BBBB22)"
+3) 5bf1fd927dfb8679496a2e6cf00cbe50c1c87145 "iPhone Developer: Profile 3 (3333CCCC33)"
+    3 valid identities found''',
+          ''
+      ));
+      mockTerminalStdInStream =
+        new Stream<String>.fromFuture(new Future<String>.error(new Exception('Cannot read from StdIn')));
```

Comment:
Hard to tell from surrounding code but it doesn't appear that this is ever used.